ipad/Constants/StringConstants.swift

```diff
@@ -69,9 +69,11 @@ struct ShiftFormHeaders {
     struct ShiftStart {
         static let startTime = "Shift Start Time *"
         static let k9OnShift = "k9 On Shift"
-        static let station = "Station"
+        static let station = "Station *"
         static let comments = "Shift Start Comments"
         static let shiftStartDate = "Shift Start Date *"
+        static let stationComments = "Station Information"
+        static let stationCommentsRequired = "Station Information *"
```

Comment:
Split the station comments in two here if required, rather than appending a "*" for better reusability